cleaners/vlc.xml

```diff
@@ -19,20 +17,68 @@
 
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+    @app VLC media player
+    @url https://www.videolan.org/vlc/index.html
+    @os Windows, Linux, FreeBSD, NetBSD, OpenBSD
+    @cleanerversion v2.0.0
+    @cleanerdate 2019-03-18.2
+    @cleanerby Andrew Ziem (2010-09-23 - 2018-10-15) & tiemay (2014-06-06) & theatre-x (2015-02-05) & ROCKNROLLKID (2017-01-14) & Vincent DUVERNET - Nolmë Informatique (???) & Tobias B. Besemer (2019-03-18)
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-10 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
+
 -->
 <cleaner id="vlc">
   <label>VLC media player</label>
   <description>Media player</description>
+  <!-- Don't know if the check of the task is really necessary... Tobias. -->
```

Comment:
This line is not necessary.

Something unintended could happen if the application is running. Speaking more broadly than VLC, some applications might fail, and in other cases they could just re-write the removed information. Either case is undesirable.

Reply:
So you preffer to have this check in every cleaner?

Reply:
Generally, yes.

Reply:
OK, I will change this in all the cleaners I (have) work(ed) on and write it down in the docu.